packages/ethereum-contracts/truffle-config.js

```diff
@@ -420,6 +435,7 @@ const E = (module.exports = {
         celoscan: process.env.CELOSCAN_API_KEY,
         basescan: process.env.BASESCAN_API_KEY,
         zkevm_polygonscan: process.env.ZKEVM_POLYGONSCAN_API_KEY,
+        scrollscan: process.env.SCROLLSCAN_API_KEY,
```

Comment:
Unfortunately this won't work unless we fork https://github.com/rkalis/truffle-plugin-verify and add support (which fortunately wouldn't be difficult though, see e.g. https://github.com/rkalis/truffle-plugin-verify/commit/b2d88609bfde2b35da60a7ac3c123a58fb79ed3c